packages/terra-core-docs/src/terra-dev-site/test/toolbar/AriaLabelToolbar.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Button from 'terra-button';
+import IconEdit from 'terra-icon/lib/icon/IconEdit';
+import IconAdd from 'terra-icon/lib/icon/IconAdd';
+import IconAttachment from 'terra-icon/lib/icon/IconAttachment';
+import Toolbar from 'terra-toolbar';
+
+export default () => (
+  <Toolbar ariaLabel="image tools">
```

Comment:
Even though the aria-label is not visible, grammatically they follow standard Title Casing similar to as if is was visible, so that it is announced in the same way to assistive technology, and will show visibly as a caption to user, so it appears the same as if it were ordinarily visible.
```suggestion
  <Toolbar ariaLabel="Image Tools">
```